system/include/emscripten/val.h

```diff
@@ -708,6 +708,7 @@ class val_awaiter {
 
 extern "C" {
   // JS FFI helper for `val_awaiter::resume_with`.
+  __attribute__((weak))
   void _emval_coro_resume(val_awaiter* awaiter, EM_VAL result) {
```

Comment:
I believe `static inline` is the more common way to do this.

Reply:
Or wait you need to export it.. ignore me,

Reply:
Ah I didn't realise it's diferent from just `inline`. I need to read up and refresh on differences...

Reply:
Nope, it doesn't work - just like with regular `inline`, it results in symbol not found.